django/db/migrations/operations/models.py

```diff
@@ -12,6 +12,16 @@
 )
 
 
+def _check_for_duplicates(arg_name, objs):
+    used_vals = set()
+    for val in objs:
+        if val in used_vals:
+            raise ValueError(
+                'Found duplicate %s in CreateModel operation.' % arg_name
```

Comment:
Is there any more complication than adding `val` in the string interpolation: `duplicate %s (%s)`? Just seems like it's a trivial addition without any downside.